package.json

```diff
@@ -82,6 +84,7 @@
     "lint-staged": "^10.0.0-beta.6",
     "ramda": "^0.26.1",
     "semantic-release": "^17.0.4",
+    "lodash": "^4.17.11",
```

Comment:
You can remove `lodash` and `@types/lodash`.

Maybe this happened because your branch is not updated.
To update your branch you can follow this instruction:
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork
or if you prefer using `rebase`:
https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a